pkg/plugin/windows/pktmon/pktmon_plugin_windows.go

```diff
@@ -0,0 +1,143 @@
+//go:build experimental
+
+package pktmon
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	v1 "github.com/cilium/cilium/pkg/hubble/api/v1"
+	kcfg "github.com/microsoft/retina/pkg/config"
+	"github.com/microsoft/retina/pkg/enricher"
+	"github.com/microsoft/retina/pkg/log"
+	"github.com/microsoft/retina/pkg/metrics"
+	"github.com/microsoft/retina/pkg/plugin/api"
+	"github.com/microsoft/retina/pkg/utils"
+	"go.uber.org/zap"
+)
+
+var (
+	ErrEnricherIsNil error = fmt.Errorf("enricher is nil")
+)
+
+const (
+	Name       = "pktmon"
+	srcIPKey   = "srcIP"
+	dstIPKey   = "dstIP"
+	srcPortKey = "srcPort"
+	dstPortKey = "dstPort"
+	protoKey   = "proto"
+)
+
+type PktMonPlugin struct {
+	enricher        enricher.EnricherInterface
+	externalChannel chan *v1.Event
+	pkt             PktMon
+	l               *log.ZapLogger
+}
+
+func (p *PktMonPlugin) Compile(ctx context.Context) error {
+	return nil
+}
+
+func (p *PktMonPlugin) Generate(ctx context.Context) error {
+	return nil
+}
```

Comment:
Why are these NOPs? Satisfying an interface?

Reply:
the plugin manager uses the plugin interface for all of the ebpf based plugins, but these windows ones don't have a use for it (at the moment, this is still an experiment)

https://github.com/microsoft/retina/blob/main/pkg/managers/pluginmanager/pluginmanager.go#L110